reporter/src/main/resources/templates/asciidoc/disclosure_document.ftl

```diff
@@ -82,12 +92,12 @@ The applicable license information is listed below:
 [#if !package.excluded]
 *Dependency*
 
-Package: [#if package.id.namespace?has_content]${package.id.namespace}:[/#if]${package.id.name}:${package.id.version}
+Package: ${packageName(package)}
 
 [#-- List the content of archived license files and associated copyrights. --]
 [#list package.licenseFiles.files as licenseFile]
 
-License File: <<${licenseFile.path}, ${licenseFile.path}>>
+License File: <<${packageName(package)} ${licenseFile.path}, ${licenseFile.path}>>
```

Comment:
These double brackets represent clickable links later in the Asciidoc PDF.
Have you tested if they still work?

Reply:
Yes I did. The part before the comma is the name of a section in the document (which now consists of the package and the file name); the part after the comma is the text to display for the link.